doc/release-notes-25412.md

```diff
@@ -0,0 +1,8 @@
+Notable changes
```

Comment:
Probably not a "notable change" (maybe the person merging the release notes will edit that, unsure).

Reply:
Done! and merged the first two commits!